carmart-tx/pom.xml

```diff
@@ -111,6 +111,12 @@
             <artifactId>infinispan-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons</artifactId>
+            <version>7.1.1.Final</version>
```

Comment:
couldn't this be replaced by ${version.org.infinispan} which should get imported from bom.